src/utils/timeUtils.js

```diff
@@ -0,0 +1,3 @@
+const formatReadTime = timeInMs => (`${Math.round(timeInMs / 60000)} min read`);
```

Comment:
This should in a helper file, not on its own.

Reply:
I was thinking this could be the helper file for everything time related, should I rename the file to something more general?